sources/packages/web/src/views/aest/institution/InstitutionDetails.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+  <h5 class="text-muted">
+    <a @click="goBack()">
+      <v-icon left> mdi-arrow-left </v-icon> Back to Institution</a
+    >
+  </h5>
+  <full-page-container>
+    <h2 color="primary-color" class="mb-15">
+      {{ initialValue.operatingName }}
+      <designation-status-badge designationStatus="&#10004; DESIGNATED" />
+    </h2>
+    <!-- TODO: Replace v-btn with vuetify2 equivalent v-tab with icon once veutify3 is released-->
+    <v-btn text variant="outlined" @click="showProfile(institutionId)"
+      ><v-icon size="25" class="mr-2">mdi-city</v-icon>Profile</v-btn
```

Comment:
just to confirm, we are planning to use vuetify3 equivalent for https://vuetifyjs.com/en/components/tabs/ in the future right?? tabs with `v-btn` will be elimated?

Reply:
Yes @ann-aot, tabs with buttons must be eliminated.
PrimeVue has the tabs component that is pretty easy to implement and will provide the exact same look and feel from Firgma.
I mentioned it to @guru-aot and I will leave it to him to decide, but I would prefer to use the one from PrimeVue.
Since we have this pending Veutify thing I will not consider this a blocker, but any developer keen to use the PrimeVue tabs now or in the upcoming task (with tabs) will have my support 😉.